docs/docs/deploy/deploy-rasa.mdx

```diff
@@ -202,7 +202,7 @@ and [specify a pull secret for the image](#step-1-image-pull-secret).
 
 #### Security Patch Releases
 Beginning with Rasa Plus 3.4.0, we release a new Docker image for supported versions each day at 0800 UTC. These images have OS security patches applied and include the date of generation in their tag in the format `YYYYMMDD`. 
-For example, to use the 3.4.2 image generated on the 1st of February 2023, use the tag `3.4.2-20230201`. These images can optionally be used as a drop-in replacement for the same version of Rasa Plus, but with all OS security updates up to that date applied.
+For example, to use the 3.4.2 image generated on the 1st of February 2023, use the tag `3.4.2-20230201`. Alternatively, you can also ensure that you always fetch the most recent security patch release using the tag format `[release]-latest` eg `3.4.2-latest`. These images can optionally be used as a drop-in replacement for the same version of Rasa Plus, but with all OS security updates up to that date applied.
```

Comment:
awesome 🙏🏻  Can I also ask you to add a link to this section in [the new Rasa Pro page](https://github.com/RasaHQ/rasa/blob/dd139604071715df6d8d04f434a975412f62dfac/docs/docs/rasa-pro.mdx?plain=1#L55)? At the bottom there's a section about "Security Scanning for Vulnerability Protection", and you'd just need to add:
```
(new line)
[Read more here](./deploy/deploy-rasa.mdx#security-patch-release).
```
(the link should be accurate but if it's not, it'll be caught by lint checks)

Reply:
Good catch, I've added that now